src/styles/block/card.css

```diff
@@ -0,0 +1,102 @@
+.container {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+}
+
+.container__card {
+  padding: 0 16px;
+}
+
+.card {
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+
+  padding-left: 3px;
+  padding-top: 32px;
+  padding-bottom: 36px;
+}
+
+.card__title {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__code {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+
+  padding-top: 4px;
+  padding-bottom: 16px;
+}
+
+.card__reviews {
+  font-weight: 400;
+  font-size: 10px;
+  color: #060b35;
+}
+
+.card__rate {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  text-align: center;
+
+  padding-bottom: 23px;
+}
+
+.card__amount {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  text-align: center;
+
+  padding-bottom: 16px;
+}
+
+.card__price {
+  font-weight: 400;
+  font-size: 12px;
+  color: #616070;
+}
+
+.card__money {
+  font-size: 16px;
+  color: #060b35;
+}
+
+.card__button-submit {
+  font-family: "Roboto", sans-serif;
+  width: 166px;
+  height: 40px;
+  border: 0;
```

Comment:
point 9 in checklist  **" Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover"**